rfd/0087/README.md

```diff
@@ -309,48 +336,12 @@ minimum that we must have:
  * the ability to output a `die` event any time a container exits
  * the die event must include the exit code
 
-As mentioned in the previous section, this problem is significantly complicated
-by the distributed nature of `attach` for docker containers in sdc-docker.
-
-### Triton Changefeed
-
-As what we'd like for the `docker run` use of `/events` is a feed of the
-changes to a given container or set of containers, the Changefeed mechanism
-described in [RFD 5](https://github.com/joyent/rfd/blob/master/rfd/0005/README.md) and
-implemented in VMAPI seems like a logical place to start. However there are a
-few things to consider here.
-
-Changefeed operates on VMAPI which is itself a cache of the VM data which
-ultimately belongs to a CN. This means that when Changefeed emits an event, it
-is telling you when the *cache* of the current state was updated, not when the
-event itself occurred on the CN. Additionally, Changefeed only tells you *that*
-something changed, not *what changed* or the new value, this means that you must
-separately query VMAPI to find the current cached state which might be different
-from the state immediately following a change since there could be intervening
-changes.
-
-As it stands currently, it's also not possible to watch changefeed for an
-individual VM. So the options would be:
-
- * modify Changefeed and allow watching for changes for an individual VM and
-   have sdc-docker create a separate listener for each `docker run` or `docker
-   attach` session.
-
- * have sdc-docker listen for *all* VM changes and filter out those we're not
-   waiting for exit from as a client of changefeed.
-
-It's open for discussion which of these would be preferable. With separate
-listeners, one concern might be the overhead and whether the Changefeed changes
-belong with the design of that API. With a single listener there would be more
-data passed that would just be ignored most of the time by sdc-docker.
-
-In addition, we'd probably want to add the `exit_status` and `boot_timestamp`
-fields to the set of VM properties that changefeed can detect.
-
-Even if we modify Changefeed in these ways, we still have a number of
-correctness problems here that seem as though they'll be impossible to solve
-with the existing implementation of changefeed. More on that in the next
-section.
+If we add support for the `die` event, we remove all the user-visible errors
+added here in the `docker run` path, and we also fix the exit codes returned by
+the Docker CLI in this case. As mentioned in the [previous
+section](#distributed-nature-of-actions-in-sdc-docker), this problem is
+significantly complicated by the distributed nature of `attach` for docker
+containers in sdc-docker.
 
 ### Correctness of exit codes and loss of information
```

Comment:
This seems like it would belong to a _separate_ section named "current limitations that don't match the solution requirements", not to a subsection of "requirements for a solution".